src/tools/clippy/clippy_lints/src/non_expressive_names.rs

```diff
@@ -117,7 +117,7 @@ impl<'a, 'tcx> SimilarNamesLocalVisitor<'a, 'tcx> {
 // this list contains lists of names that are allowed to be similar
 // the assumption is that no name is ever contained in multiple lists.
 #[rustfmt::skip]
-const WHITELIST: &[&[&str]] = &[
+const EXEMPTIONS: &[&[&str]] = &[
```

Comment:
if you renamed this (and the related occurrence of `exemptions` above) to `ALLOWED_TO_BE_SIMILAR` (and `allowed_to_be_similar`) then the code would be more self-documenting.

Reply:
(such a rename may require some extra line-breaking to placate tidy, but I personally would be okay with that.)

Reply:
Good point; done.